### PR TITLE
Added float32raw and float64raw to CellType.fromString

### PR DESCRIPTION
**Pull request by jasonbeverage**

### CODE REVIEW

**Comment by moradology** [comment]
looks good, +1

File: raster/src/main/scala/geotrellis/raster/CellType.scala [hunk 1]
```diff
@@ -206,6 +206,8 @@ object CellType {
     case "int32" => IntConstantNoDataCellType
     case "float32" => FloatConstantNoDataCellType
     case "float64" => DoubleConstantNoDataCellType
+    case "float32raw" => FloatCellType
```
Comment (moradology) [inline raster/src/main/scala/geotrellis/raster/CellType.scala]: Would you mind moving this up to match the placement of 'raw' strings above?

**Comment by moradology** [comment]
+1 after minor stylistic gripe

**Comment by jasonbeverage** [comment]
Sure no problem, just moved them.

**Pull request merged**